chapters/appendix-VI-external-repository-identifiers.md

```diff
@@ -133,12 +130,51 @@ External Reference Site: [https://github.com/package-url/purl-spec](https://gith
 
 Documentation: [https://github.com/package-url/purl-spec](https://github.com/package-url/purl-spec)
 
-***
-When <category> = `OTHER`:
-***
 
-## `<type>` [idstring] <a name="idstring"></a>
+## When `<category>` = `PERSISTENT-ID`:
+
+### `<type>` swh <a name="swh"></a>
+
+#### `<locator>` Information
+
+These point to objects present in the Software Heritage archive by the means of persistent identifiers
+that are guaranteed to remain stable (persistent) over time.
+Their syntax is described below.
+Note that they are identifiers and not URLs.
+
+A persistent identifier can point to any software artifact (or “object”)
+available in the Software Heritage archive.
+Objects come in different types, and most notably:
+
+* contents
```

Comment:
Should we add the short form from the grammer in the description?  either "cnt - contents" or "contents (cnt)"?

Reply:
I've put them in the right order between description and grammar. ;-)
I tried to not overload the document with details, although providing more info than the minimal CPE types. I think linking to external documentation covers it.

Reply:
Agree with @zvr here, its following the pattern with the others, so lets go forward with this as is right now, and revisit how its described in 3.0.